test/fork/dao.test.js

```diff
@@ -357,4 +373,179 @@ describe('Integration tests on Gnosis deployment', () => {
+    await Promise.all(TOKEN_HOLDERS_ADDRESSES.map((_address) => sendEth(hre.ethers, faucet, _address, '10')))
+    await missingSteps()
+  })
+
+  it('should process pegout and withdrawInflation', async () => {
+    const metadata = encode(
+      ['bytes1', 'bytes', 'bytes4', 'address', 'bytes4', 'address', 'bytes', 'bytes'],
+      [
+        '0x02',
+        // secretlint-disable-next-line
+        '0x00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000cf759bccfef5f322af58adae2d28885658b5e0200000000000000000000000000000000000000000000000000000000000003a0000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000003000000000000000000000000f4ea6b892853413bd9d9f1a5d3a620a0ba39c5b2000000000000000000000000f4ea6b892853413bd9d9f1a5d3a620a0ba39c5b2000000000000000000000000e396757ec7e6ac7c8e5abe7285dde47b98f22db80000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000e0000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000000443352d49b0000000000000000000000000123456789012345678901234567890123456789000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044095ea7b3000000000000000000000000e396757ec7e6ac7c8e5abe7285dde47b98f22db8000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000124c322525d000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000f4ea6b892853413bd9d9f1a5d3a620a0ba39c5b200000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000010000f1918e00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002a30783632333939363865363233313136343638374342343066383338396439333364443766376530413500000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'.replaceAll(
+          '0123456789012345678901234567890123456789',
+          (await forwarderNativePermissioned.getAddress()).slice(2)
+        ),
+        PNETWORK_NETWORK_IDS.gnosisMainnet,
+        ZERO_ADDRESS,
+        PNETWORK_NETWORK_IDS.ethereumMainnet,
+        await forwarderNativePermissioned.getAddress(),
+        '0x',
+        '0x'
+      ]
+    )
+    await expect(
+      vault.connect(pnetwork).pegOut(await forwarderNativePermissioned.getAddress(), PNT_ON_ETH_ADDRESS, 1, metadata)
```

Comment:
Seeing the usage throughout the test, why not storing it in a constant?
```suggestion
      vault.connect(pnetwork).pegOut(forwarderNativeAddress, PNT_ON_ETH_ADDRESS, 1, metadata)
```

Reply:
I'll be using `forwarderNativePermissioned.target` which is not async in an upcoming refactoring